src/lib/components/ThemeToggle.svelte

```diff
@@ -1,54 +1,40 @@
 <script lang="ts">
-  import Toggle from "./Toggle.svelte";
   import { Theme } from "$lib/types/theme";
   import { themeStore } from "$lib/stores/theme.store";
   import { i18n } from "$lib/stores/i18n";
   import IconLightMode from "$lib/icons/IconLightMode.svelte";
   import IconDarkMode from "$lib/icons/IconDarkMode.svelte";
 
-  const switchTheme = ({ detail }: CustomEvent<boolean>) =>
-    themeStore.select(detail ? Theme.DARK : Theme.LIGHT);
+  const switchTheme = () => {
+    themeStore.select($themeStore === Theme.LIGHT ? Theme.DARK : Theme.LIGHT);
+  };
 
-  let checked: boolean;
-  $: checked = $themeStore !== Theme.LIGHT;
+  $: isDarkMode = $themeStore === Theme.DARK;
```

Comment:
For consistency reason can you declare the variable.

```
let darkMode: boolean;
$: darkMode = $themeStore
```

NTH: I might also avoid naming it 'isSomething'. We generally prefer being expressive in our variable names, but I could be wrong.

Reply:
Not sure about the naming conventions here but to me darkMode and isDarkMode means two different things.

Reply:
it looks like it is used in just one other location, so not really common naming convention. Do you want me to change it?

https://github.com/dfinity/gix-components/blob/NNS1-2965-update-theme-toggle/src/lib/components/Tooltip.svelte#L28

Reply:
Naming was a NTH, works for me.

Declaring the variable would be consistent.

Reply:
I agree with Cosku, that `isDarkMode` is clearer. With `darkMode` I wouldn't be sure about the type but with `isDarkMode` it's obviously a `boolean`.

Reply:
Variable added.

Reply:
We tend to use `isSomething` for function and `something` for variable, there is not linter and it's just a feeling so for sure it's fine.